src/Reset.ts

```diff
@@ -225,10 +290,6 @@ export const reset = (input: resetNames, fast = false, from = 'unknown') => {
 
     if (input === "transcension" || input === "transcensionChallenge" || input == "reincarnation" || input == "reincarnationChallenge"
         || input === "ascension" || input === "ascensionChallenge") {
```

Comment:
```suggestion
    const types = ['transcension', 'transcensionChallenge', 'reincarnation', 'reincarnationChallenge', 'ascension', 'ascensionChallenge'];
    if (types.includes(input)) {
```

Reply:
I left this one alone because I didn't touch the code, but I might as well see if GitHub's "commit suggestion" button works